scripts/migrateServerDB/index.ts

```diff
@@ -1,38 +1,91 @@
+// Force server mode for migration
+process.env.NEXT_PUBLIC_SERVICE_MODE = 'server';
+
 import * as dotenv from 'dotenv';
-import * as migrator from 'drizzle-orm/neon-serverless/migrator';
+import { drizzle } from 'drizzle-orm/node-postgres';
+import * as migrator from 'drizzle-orm/node-postgres/migrator';
 import { join } from 'node:path';
+import { Pool } from 'pg';
 
-import { serverDB } from '../../src/database/server/core/db';
 import { PGVECTOR_HINT } from './errorHint';
 
-// Read the `.env` file if it exists, or a file specified by the
-// dotenv_config_path parameter that's passed to Node.js
+// Read the `.env` file if it exists
 dotenv.config();
 
 const runMigrations = async () => {
-  await migrator.migrate(serverDB, {
-    migrationsFolder: join(__dirname, '../../src/database/migrations'),
+  const connectionString = process.env.DATABASE_URL;
+  if (!connectionString) {
+    console.log(' not find database env, migration skipped');
+    process.exit(0);
+  }
+
+  const pool = new Pool({ 
+    connectionString,
+    ssl: false,
   });
-  console.log('✅ database migration pass.');
+  
+  const db = drizzle(pool);
+
+  try {
+    // Mark the problematic migration as completed since the column already exists
+    const existingMigrations = [
```

Comment:
it's not a good idea to hard code migrations, it will make much more maintenance efforts. So it's better to just read from the migrationsFolder